testing/factory/factory.go

```diff
@@ -679,3 +679,13 @@ func (factory *Factory) MakeSpace(c *gc.C, params *SpaceParams) *state.Space {
 	c.Assert(err, jc.ErrorIsNil)
 	return space
 }
+
+func (factory *Factory) CurrentCfg(c *gc.C) *config.Config {
```

Comment:
Needs a comment if exported.
Should be called simply ModelConfig()
Is it called elsewhere other than in here? If not, doesn't need to be exported.